app/policies/program_policy.rb

```diff
@@ -1,6 +1,14 @@
 class ProgramPolicy < ApplicationPolicy
+  delegate :create?, to: :program_policy
+
   def new?
-    Pundit.policy_scope!(user, Organization).present?
+    can_access_at_least_one_organization?
+  end
+
+  private
+
+  def program_policy
```

Comment:
It seems weird to me that this is called `program_policy` when it's returning an instance of `OrganizationPolicy`.

Reply:
Good catch.